### ISSUE TITLE
� PINN — 2026-04-04 (43 papers)

### ISSUE THREAD

**Issue by DaehyeopKim** [DaehyeopKim/DaehyeopKim.github.io#45]
### 🏆 Oral

**1. [PHYSICS-INFORMED RADIAL BASIS FUNCTION NETWORKS AND KOLMOGOROV-ARNOLD NETWORKS](https://openreview.net/forum?id=wBz7H1xBbV)**
  *Стенькин Дмитрий Александрович, Горбаченко Владимир Иванович*
  📍 MathAI 2025 Oral
  > Physics-informed neural networks are trained by minimizing the loss function, which is the sum of the squares of the residuals of the equation or system of equations being solved. Such networks do ...

### ⭐ Spotlight

**2. [RoboScape: Physics-informed Embodied World Model](https://openreview.net/forum?id=Jo8PhYaCFt)**
  *Yu Shang, Xin Zhang, Yinzhou Tang et al.*
  📍 CoRR 2025
  > World models have become indispensable tools for embodied intelligence, serving as powerful simulators capable of generating realistic robotic videos while addressing critical data scarcity challen...

**3. [UPS: Efficiently Building Foundation Models for PDE Solving via Cross-Modal Adaptation](https://openreview.net/forum?id=HctgQ6UKee)**
  *Junhong Shen, Tanya Marwah, Ameet Talwalkar*
  📍 ICML2024-AI4Science Spotlight
  > We present Unified PDE Solvers (UPS), a data- and compute-efficient approach to developing unified neural operators for diverse families of spatiotemporal PDEs from various domains, dimensions, and...

### 📌 Poster

**4. [Equation identification for fluid flows via physics-informed neural networks](https://openreview.net/forum?id=XsvCLEYH3O)**
  *Alexander New, Marisel Villafañe-Delgado, Charles Shugert*
  📍 ICML2024-AI4Science Poster
  > Scientific machine learning (SciML) methods such as physics-informed neural networks (PINNs) are used to estimate parameters of interest from governing equations and small quantities of data. Howev...

**5. [Fourier Neural Operator based surrogates for $\textrm{CO}_2$ storage in realistic geologies](https://openreview.net/forum?id=DBHG4c9F6h)**
  *Anirban Chandra, Marius Koch, Suraj Pawar et al.*
  📍 ICML2024-AI4Science Poster
  > This study aims to develop surrogate models for accelerating decision making processes associated with carbon capture and storage (CCS) technologies.  Selection of sub-surface $\textrm{CO}_2$ stora...

**6. [ASNO: An Interpretable Attention-Based Spatio-Temporal Neural Operator for Robust Scientific Machine Learning](https://openreview.net/forum?id=RkCBoYwWDh)**
  *Vispi Nevile Karkaria, Doksoo Lee, Yi-Ping Chen et al.*
  📍 ICML 2025 R2-FM Workshop Poster
  > Scientific machine learning (SciML) aims to model complex physical processes from data, but a key challenge lies in capturing long-range spatio-temporal dependencies while generalizing across varyi...

**7. [SepONet: Efficient Large-Scale Physics-Informed Operator Learning](https://openreview.net/forum?id=7EK92XZ5L5)**
  *Xinling Yu, Sean Hooten, Ziyue Liu et al.*
  📍 D3S3 2024 Poster
  > We introduce Separable Operator Networks (SepONet), a novel framework that significantly enhances the efficiency of physics-informed operator learning. SepONet uses independent trunk networks to le...

**8. [Consistency of Physics-Informed Neural Networks for Second-Order Elliptic Equations](https://openreview.net/forum?id=zL4JRfBr7R)**
  *Yuqian Cheng, Zhuo Chen, Qian Lin*
  📍 NeurIPS 2025 poster
  > The physics-informed neural networks (PINNs) are widely applied in solving differential equations. However, few studies have discussed their consistency. In this paper, we consider the consistency ...

**9. [Physics-Informed Shearlet Neural Operator (PI-ShearletNO) for parametric partial differential equations](https://openreview.net/forum?id=sMT90FMPoi)**
  *Fabio Pereira dos Santos, Júlio de Castro Vargas Fernandes, Adriano M A Cortes*
  📍 AI&PDE Poster
  > This paper introduces the Physics-Informed Shearlet Neural Operator (PI-ShearletNO), a framework for learning solution operators of parametric partial differential equations. The model combines neu...

**10. [Harmonized Cone for Feasible and Non-conflict Directions in Training Physics-Informed Neural Networks](https://openreview.net/forum?id=PRYl1mO1go)**
  *Dohyun Bu, Yujung Byun, Jong-Seok Lee*
  📍 ICLR 2026
  > Physics-Informed Neural Networks (PINNs) have emerged as a powerful tool for solving PDEs, yet training is difficult due to a multi-objective loss that couples PDE residuals, initial/boundary condi...

**11. [Fast Convergence of Natural Gradient Descent for Over-parameterized Physics-Informed Neural Networks](https://openreview.net/forum?id=KWWfLgkySm)**
  *Xianliang Xu, Wang Kong, JiahengM et al.*
  📍 ICLR 2026
  > In the context of over-parameterization, there is a line of work demonstrating that randomly initialized (stochastic) gradient descent (GD) converges to a globally optimal solution at a linear conv...

**12. [Enhancing Stability of Physics-Informed Neural Network Training Through Saddle-Point Reformulation](https://openreview.net/forum?id=EQNp3sFrY3)**
  *Dmitry Bylinkin, Mikhail Aleksandrov, Savelii Chezhegov et al.*
  📍 ICLR 2026
  > Physics-informed neural networks (PINNs) have gained prominence in recent years and are now effectively used in a number of applications. However, their performance remains unstable due to the comp...

**13. [MAD-PINN: A Decentralized Physics-Informed Machine Learning Framework for Safe and Optimal Multi-Agent Control](https://openreview.net/forum?id=nwcFuR19AJ)**
  *Manan Tayal, Aditya Singh, Shishir Kolathaya et al.*
  📍 AAAI 2026
  > Co-optimizing safety and performance in large-scale multi-agent systems remains a fundamental challenge. Existing approaches based on multi-agent reinforcement learning (MARL), safety filtering, or...

### 🆕 Recent (ArXiv / OpenReview)

**14. [Multifidelity modeling for Physics-Informed Neural Networks (PINNs)](https://openreview.net/forum?id=z9jSS8VEcR)**
  *Michael Penwarden, Shandian Zhe, Akil Narayan et al.*
  📍 J. Comput. Phys. 2022
  > Highlights•Low-rank multifidelity construction of surrogate models for Physics-Informed Neural Networks.•Width, depth, and optimization hyperparameters of Physics-Informed Neural Networks as a form...

**15. [Physics-Informed Neural Operators with Multi-Method Explainability for Ocean Temperature Prediction](https://openreview.net/forum?id=ZT9llBabR5)**
  *Shivam Dubey*
  📍 XAI4Science Workshop 2026
  > Accurate ocean temperature prediction is critical for climate modeling and marine ecosystem management, yet current machine learning approaches often lack physical consistency and interpretability....

**16. [Physics-Informed Neural Operator for Coupled Forward-Backward Partial Differential Equations](https://openreview.net/forum?id=iLwfzf33Ub)**
  *Xu Chen, Yongjie FU, Shuo Liu et al.*
  📍 SynS & ML @ ICML2023
  > This paper proposes a physics-informed neural operator (PINO) framework to solve a system of coupled forward-backward partial differential equations (PDEs) arising from mean field games (MFGs). The...

**17. [Good Lattice Accelerates Physics-Informed Neural Networks](https://openreview.net/forum?id=eqOWfGGo7Z)**
  *Takashi Matsubara, Takaharu Yaguchi*
  📍 SynS & ML @ ICML2023
  > Physics-informed neural networks (PINNs) can solve partial differential equations (PDEs) by minimizing the physics-informed loss, ensuring the neural network satisfies the PDE at given points. Howe...

**18. [Data and Modeling Assumptions in Physics-Informed Operator Learning](https://openreview.net/forum?id=C5sL9W8kwY)**
  *Martin Hofmann-Wellenhof, Alexander Fuchs, Bernhard C Geiger et al.*
  📍 DiffSys 2025
  > Operator networks have emerged as promising surrogate models, replacing computationally expensive numerical solvers for differential equations. Beyond achieving competitive accuracy with traditiona...

**19. [Zero coordinate shift: Whetted automatic differentiation for physics-informed operator learning](https://openreview.net/forum?id=4zZFDAGDV0)**
  *Kuangdai Leng, Mallikarjun Shankar, Jeyan Thiyagalingam*
  📍 J. Comput. Phys. 2024
  > Highlights•We present a novel algorithm to conduct automatic differentiation w.r.t. coordinates for physics-informed operator learning.•Our algorithm can reduce GPU memory and wall time for trainin...

**20. [A Taxonomic Survey of Physics-Informed Machine Learning](https://openreview.net/forum?id=IuFbkVxfmR)**
  *Preetam Ghosh, Joseph Pateras, Pratip Rana*
  📍 Crossref
  > Physics-informed machine learning (PIML) refers to the emerging area of extracting physically relevant solutions to complex multiscale modeling problems and has enjoyed significant interest from th...

**21. [Label Propagation Training Schemes for Physics-Informed Neural Networks and Gaussian Processes](https://openreview.net/forum?id=L4Mbudj3le)**
  *Ming Zhong, Dehao Liu, Raymundo Arróyave et al.*
  📍 CoRR 2024
  > This paper proposes a semi-supervised methodology for training physics-informed machine learning methods. This includes self-training of physics-informed neural networks and physics-informed Gaussi...

**22. [Physics-informed neural networks for pathloss prediction](https://openreview.net/forum?id=AhBkIqK3nC)**
  *Steffen Limmer*
  📍 MLSP2023
  > This paper introduces a physics-informed machine learning approach for pathloss prediction. This is achieved by including in the training phase simultaneously (i) physical dependencies between spat...

**23. [Revisiting Conservativeness in Fluid Dynamics: Failure of Non-Conservative PINNs and a Path-Integral Remedy](http://arxiv.org/abs/2604.01968v1)**
  *Arun Govind Neelan, Ferdin Sagai Don Bosco, Naveen Sagar Jarugumalli et al.*
  > The choice between conservative and non-conservative formulations is a fundamental dilemma in CFD. While non-conservative forms offer intuitive modeling in primitive variables, they typically produ...

**24. [Goal oriented error estimation for adaptive sampling of PINNS](http://arxiv.org/abs/2604.01835v1)**
  *Medard Govoeyi, Thomas Richter*
  > Physics-Informed Neural Networks (PINNs) are mesh-free approaches for the numerical approximation of partial differential equations, where a neural network is trained by minimizing a loss function ...

**25. [Mixed Consistent PINNs for Elliptic Obstacle Problems with Stability Analysis](http://arxiv.org/abs/2604.01719v1)**
  *Arbaz Khan, Kent-Andre Mardal, Shiv Mishra*
  > We propose a consistent physics-informed neural networks (CPINNs) framework for elliptic obstacle problems formulated as variational inequalities. The method is based on a mixed loss functional tha...

**26. [Inverting the Kohn–Sham equations with physics-informed machine learning](https://openreview.net/forum?id=yBwSbFyqsT)**
  *Vincent Martinetto, Karan Shah, Aurora Pribram-Jones et al.*
  📍 Machine Learning Science & Technology
  > Electronic structure theory calculations offer an understanding of matter at the quantum level, complementing experimental studies in materials science and chemistry. One of the most widely used me...

**27. [Physics-informed kernel learning](https://openreview.net/forum?id=woVEU0Bm4H)**
  *Nathan Doumèche, Francis R. Bach, Gérard Biau et al.*
  📍 CoRR 2024
  > Physics-informed machine learning typically integrates physical priors into the learning process by minimizing a loss function that includes both a data-driven term and a partial differential equat...

**28. [Physics-informed machine learning for moving load problems](https://openreview.net/forum?id=w9Lm0yAfG8)**
  *Taniya Kapoor, Hongrui Wang, Alfredo Núñez et al.*
  📍 CoRR 2023
  > This paper presents a new approach to simulate forward and inverse problems of moving loads using physics-informed machine learning (PIML). Physics-informed neural networks (PINNs) utilize the unde...

**29. [Physics-Informed Model-Based Reinforcement Learning](https://openreview.net/forum?id=9pTYpmtfU8)**
  *Adithya Ramesh, Balaraman Ravindran*
  📍 CoRR 2022
  > We apply reinforcement learning (RL) to robotics tasks. One of the drawbacks of traditional RL algorithms has been their poor sample efficiency. One approach to improve the sample efficiency is mod...

**30. [Physics-Informed Learning of Characteristic Trajectories for Smoke Reconstruction](https://openreview.net/forum?id=EwqEtMSgOp)**
  *Yiming Wang, Siyu Tang, Mengyu Chu*
  📍 SIGGRAPH (Conference Paper Track) 2024
  > We delve into the physics-informed neural reconstruction of smoke and obstacles through sparse-view RGB videos, tackling challenges arising from limited observation of complex dynamics. Existing ph...

**31. [Evaluating Deep Surrogate Models for Knee Joint Contact Mechanics Under Input-Limited Conditions](http://arxiv.org/abs/2604.01990v1)**
  *Zhengye Pan, Jianwei Zuo, Jiajia Luo*
  > Background and Objective: Accurate surrogate modeling of knee joint contact mechanics is important for reconstructing stress distributions and identifying risk-relevant regions, yet the relative su...

**32. [Physics-Informed Transformer for Multi-Band Channel Frequency Response Reconstruction](http://arxiv.org/abs/2604.01944v1)**
  *Anatolij Zubow, Joana Angjo, Sigrid Dimce et al.*
  > Wideband channel frequency response (CFR) estimation is challenging in multi-band wireless systems, especially when one or more sub-bands are temporarily blocked by co-channel interference. We pres...

**33. [PDEBench: An Extensive Benchmark for Scientific Machine Learning](https://openreview.net/forum?id=pjMgz7AZG8)**
  *Makoto Takamoto, Timothy Praditia, Raphael Leiteritz et al.*
  📍 NeurIPS 2022
  > Machine learning-based modeling of physical systems has experienced increased interest in recent years. Despite some impressive progress, there is still a lack of benchmarks for Scientific ML that ...

**34. [Pi-fusion: Physics-informed diffusion model for learning fluid dynamics](https://openreview.net/forum?id=FsTY5AbW7M)**
  *Jing Qiu, Jiancheng Huang, Xiangdong Zhang et al.*
  📍 CoRR 2024
  > Physics-informed deep learning has been developed as a novel paradigm for learning physical dynamics recently. While general physics-informed deep learning methods have shown early promise in learn...

**35. [Optimisation of quantitative brain diffusion-relaxation MRI acquisition protocols with physics-informed machine learning](https://openreview.net/forum?id=64GyZYTJkt)**
  *Álvaro Planchuelo-Gómez, Maxime Descoteaux, Hugo Larochelle et al.*
  📍 Medical Image Anal. 2024
  > Highlights•Physics-informed networks were compared to manual selection and CRLB optimisation.•Physics-informed approaches identified subsets that yielded consistently accurate estimates.•Five-fold ...

**36. [Physics-Informed Spatio-Temporal Model for Human Mobility Prediction](https://openreview.net/forum?id=YG0Km0K5dT)**
  *Quanyan Gao, Chao Li, Qinmin Yang*
  📍 ECML/PKDD (2) 2024
  > Human mobility prediction is the fundamental problem in studying human social behaviors. However, current approaches overlook the dynamic physics processes inherent in the movement of human, especi...

**37. [gLaSDI: Parametric physics-informed greedy latent space dynamics identification](https://openreview.net/forum?id=RMEYsjtnAW)**
  *Xiaolong He, Youngsoo Choi, William D. Fries et al.*
  📍 J. Comput. Phys. 2023
  > Highlights•Non-intrusive reduced-order modeling based on physics-informed greedy latent-space dynamics identification is proposed.•An autoencoder and parametric models are trained interactively to ...

**38. [Loop-level surrogate modeling of dopant-distribution effects in Ba(Zr,Ti)O$_3$](http://arxiv.org/abs/2604.02325v1)**
  *Heiko Röthl, Elke Kraker, Julien Magnien et al.*
  > Barium titanate-based perovskites are important candidates for lead-free dielectric and electromechanical technologies. In Zr-substituted BaTiO$_3$ (BZT), functional behavior is usually discussed i...

**39. [RMDM: Radio Map Diffusion Model with Physics Informed](https://openreview.net/forum?id=yRlExvLwfI)**
  *Haozhe Jia, Wenshuo Chen, Zhihui Huang et al.*
  📍 CoRR 2025
  > With the rapid development of wireless communication technology, the efficient utilization of spectrum resources, optimization of communication quality, and intelligent communication have become cr...

**40. [A Mean-Field Game Model For Large-Scale Attrition in Attacker-Defender Systems](http://arxiv.org/abs/2604.02101v1)**
  *Avetik Arakelyan, Tigran Bakaryan, Davit Alaverdyan et al.*
  > This paper proposes a novel Mean-Field Game (MFG) framework for large-scale attacker-defender systems aimed at protecting one or multiple High-Value Units (HVUs). Motivated by classical agent-wise ...

**41. [Physics Informed RNN-DCT Networks for Time-Dependent Partial Differential Equations](https://openreview.net/forum?id=eFUoB1mbqa)**
  *Benjamin Wu, Oliver Hennigh, Jan Kautz et al.*
  📍 ICCS (1) 2022
  > Physics-informed neural networks allow models to be trained by physical laws described by general nonlinear partial differential equations. However, traditional architectures of this approach strug...

**42. [PDEformer-2: A Versatile Foundation Model for Two-Dimensional Partial Differential Equations](https://openreview.net/forum?id=3BSl8shjea)**
  *Zhanhong Ye, Zining Liu, Bingyang Wu et al.*
  📍 CoRR 2025
  > Partial differential equations (PDEs) play a central role in describing many physical phenomena. Various scientific and engineering applications demand a versatile and differentiable PDE solver tha...

**43. [PDETime: Rethinking Long-Term Multivariate Time Series Forecasting from the perspective of partial differential equations](https://openreview.net/forum?id=DMzycQs5BW)**
  *Shiyi Qi, Zenglin Xu, Yiduo Li et al.*
  📍 CoRR 2024
  > Recent advancements in deep learning have led to the development of various models for long-term multivariate time-series forecasting (LMTF), many of which have shown promising results. Generally, ...
